### DISCUSSION TITLE
Welcome to nexus-chain Discussions!

### DISCUSSION THREAD

**Discussion by KOSASIH** [KOSASIH/nexus-chain#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by locksparty** [comment]
Hello , I'm unable to build using GO

**Comment by Clawue884** [comment]
Saya adalah penggemar anda pak kosasih saya berasal dari cilacap dan saya sangat takjub dengan projek anda semoga berjalan lancar dan bermanfaat
